docs/multicluster/architecture.md

```diff
@@ -201,3 +200,25 @@ be created as a ResourceExport in the leader cluster, and the resource
 export/import pipeline will ensure member clusters receive this ACNP spec
 to be replicated. Each member cluster's Multi-cluster Controller will then
 create an ACNP in their respective clusters.
+
+## Antrea Multi-cluster Traffic Modes
+
+Multi-cluster Gateway supports all of `encap`, `noEncap`, `hybrid`, and
+`networkPolicyOnly` modes. In all supported modes, the cross-cluster traffic
+is routed by Multi-cluster Gateways of member clusters, and the traffic goes
+through Antrea overlay tunnels between Gateways. The difference between
+`networkPolicyOnly` and others modes is Antrea is no longer the primary CNI
```

Comment:
This sentence sounds like Antrea is no longer primary CNI with Multi-cluster, which is not correct. I do not feel we should emphasize primary CNI / chaining either, as that is part of networkPolicyOnly mode, nothing specific to Multi-cluster.

If you want to talk about the difference from encap mode, I feel you should say: "In noEncap, hybrid, and networkPolicyOnly modes, even when in-cluster Pod traffic does not go through tunnels, `antrea-agent` still creates tunnels between the Gateway Node and other Nodes, and routes cross-cluster traffic to reach the Gateway through the tunnels."

And if you really like to mention CNI chaining, etc., say: "Specially for [networkPolicyOnly mode](../design/policy-only.md), Antrea only handles multi-cluster traffic routing, while the primary CNI takes care of in-cluster traffic routing".

Reply:
Done